### PR TITLE
Contact me feature

### PR DESCRIPTION
**Pull request by wassimchakib**
In this feature, I added : 
• Created a form contact that contains : 
 1. Input text for the full name
 2. Input text for the Email
 3. Textarea for the message
 4. Send the form into formspree

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -282,6 +283,74 @@ body {
...
+    padding-top: 114px;
+  }
+
+  #contact-form {
+    background-image: url(/images/contact-form.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+  }
```
Comment (leonardodiasb) [inline style.css]: - [ ] The form style is excellent, but on some mobile screens, the background image gets overlapped with the white text making it hard to read. I suggest you shrink the background image a little so the white semicircle isn't overlapped by the text.
 -Your design:
image

-Figma design:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @wassimchakib, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of GitHub flow. ✔️
 - Followed Figma design. ✔️ 
 - Professional PR. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GiuseppeTG — APPROVED** [review]
Hi @wassimchakib  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights
✅ Professional PR
✅ Good job implementing the changes required
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GiuseppeTG** [comment]
[OPTIONAL] I will mark this as optional but it should have been requested before: please follow the Figma design for button interaction as they are not the same as in the template. Especially when hover and pressed 🤓 
The rest is looking very good, congrats!

Figma:
Captura de Pantalla 2022-09-27 a la(s) 7 09 59 a  m

**Pull request merged**